### PR TITLE
removed <BetaMessage /> to no longer display user-panel info

### PR DESCRIPTION
**Pull request by georgiaewhitney**
## Who is this for?
Part of #10163 

## What is it doing for them?
Removes the `<BetaMessage>` component / div displaying info about the user panel as we're not currently collecting this data. 

Entire component removed as to get rid of the 'tools logo' and keep the panel clean

New:
Screenshot 2023-08-31 at 17 17 08

Previously:
Screenshot 2023-08-31 at 17 17 38

### CODE REVIEW

File: content/webapp/pages/concepts/[conceptId].tsx [hunk 1]
```diff
@@ -342,21 +341,9 @@ export const ConceptPage: NextPage<Props> = ({
...
-                    or{' '}
-                    <Link href="https://roadmap.wellcomecollection.org/">
-                      submit an idea
-                    </Link>{' '}
-                    that would help you use our website.
-                  </>
-                }
-              />
-            </ConceptDescription>
+            <ConceptDescription
```
Comment (gestchild) [inline content/webapp/pages/concepts/[conceptId].tsx:344]: I think this (and the related styles) can be removed too

Comment (georgiaewhitney) [reply content/webapp/pages/concepts/[conceptId].tsx:344]: Ah yes - thanks!!

**Comment by gestchild** [comment]
You can remove this too now: https://github.com/wellcomecollection/wellcomecollection.org/blob/6a4a86b0e3dc8653e6b401b20311d5f782715284/content/webapp/pages/concepts/%5BconceptId%5D.tsx#L84

**Review by gestchild — APPROVED** [review]
✨

**Pull request merged**